src/python/pants/backend/jvm/artifact.py

```diff
@@ -27,10 +25,10 @@ def __init__(self, org, name, repo, publication_metadata=None):
     :param repo: The ``repo`` this artifact is published to.
     :param publication_metadata: Optional extra publication metadata required by the ``repo``.
     """
-    if not isinstance(org, string_types):
-      raise ValueError("org must be {} but was {}".format(string_types, org))
-    if not isinstance(name, string_types):
-      raise ValueError("name must be {} but was {}".format(string_types, name))
+    if not isinstance(org, str):
+      raise ValueError("org must be str but was {}".format(org))
```

Comment:
We can use fstrings now, no? Or should we discuss this first as a thing to allow?

Reply:
Yes, we can! The only concern is if it's likely that we will need to cherry-pick into 1.17.x (which I think is close to the final cut, after which we can liberally use Py3 features like type hints).

This particular block, and the others changed, seem highly unlikely to need to be cherry-picked so I'll make the update.